theLudovyc/GUI/ResourceControl.gd

```diff
@@ -4,11 +4,13 @@ extends PanelContainer
 
 @onready var label_amount := $HBoxContainer/PanelContainer/HBoxContainer/LabelAmount
 @onready var label_production_rate := $HBoxContainer/PanelContainer/HBoxContainer/LabelProductionRate
+@onready var texture_button: ResourceButton = $TextureButton
```

Comment:
You can use ":= value" it is same as ":Type = value". Please see https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#variables

Reply:
This static typing help with the experience in the IDE.
https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/static_typing.html

I am aware that you do not need to declare types in GDScript but it helps with the autocompletion in the IDE (Rider in my case). Also i am able to navigate between methods faster this way.

Reply:
See https://github.com/unknown-horizons/godot-port/pull/124#discussion_r1847315092